sdk/storage/azure-storage-file-datalake/tests/test_file_async.py

```diff
@@ -875,9 +1039,10 @@ async def test_rename_file_to_existing_file_async(self, datalake_storage_account
 
         data = await (await new_client.download_file()).readall()
         # the existing file was overridden
-        self.assertEqual(data, data_bytes)
+        assert data == data_bytes
 
     @DataLakePreparer()
+    @recorded_by_proxy_async
     async def test_file_encryption_scope_from_file_system_async(self, datalake_storage_account_name,
```

Comment:
Script didn't catch these params since they are on multiple lines. Can you update this to the `kwargs` approach manually?